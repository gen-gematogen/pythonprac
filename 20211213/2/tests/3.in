[152, 157, 229, 136, 61, 1854, 271, 5136, 124, 176, 1351, 197, 221, 245, 72, 125, 1062, 282, 535, 169, 692, 173, 59, 175, 263, 486, 137, 289, 2281, 262, 1074, 128, 78, 90, 2423, 67, 191, 289, 2530, 170, 178, 79, 198, 13267, 281, 275, 187, 18317, 27, 114, 244, 225, 1662, 60, 153, 186, 297, 1140, 178, 39, 177, 279, 277]
