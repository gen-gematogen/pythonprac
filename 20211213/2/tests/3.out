[27, 39, 59, 60, 61, 67, 72, 78, 79, 90, 114, 124, 125, 128, 136, 137, 152, 153, 157, 169, 170, 173, 175, 176, 177, 178, 178, 186, 187, 191, 197, 198, 221, 225, 229, 244, 245, 262, 263, 271, 275, 277, 279, 281, 282, 289, 289, 297, 486, 535, 692, 1062, 1074, 1140, 1351, 1662, 1854, 2281, 2423, 2530, 5136, 13267, 18317]
